models/renalStructures_UNEST_segmentation/docs/README.md

```diff
@@ -1,7 +1,9 @@
 # Description
 A pre-trained model for inferencing volumetric (3D) kidney substructures segmentation from contrast-enhanced CT images (Arterial/Portal Venous Phase).
+
```

Comment:
I think this blank line is not necessary.

Thanks.

Reply:
Thanks, deleted.